pydatastructs/strings/tests/test_trie.py

```diff
@@ -2,14 +2,21 @@
 
 def test_Trie():
 
-    strings = ["A", "to", "tea", "ted", "ten", "i", "in", "inn"]
+    strings = ["A", "to", "tea", "ted", "ten", "i",
+               "in", "inn", "Amfn", "snbr"]
     trie = Trie()
     for string in strings:
         trie.insert(string)
 
+    prefix_strings = ["te", "t", "Am", "snb"]
+
     for string in strings:
         assert trie.is_present(string)
 
+    for string in prefix_strings:
+        assert trie.is_prefix_present(string)
+        assert not trie.is_present(string)
```

Comment:
Well, all the strings in `prefix_strings` are there in the trie. They are not absent, they are just present as prefixes of other strings.

Reply:
I think what you are trying to do can be achieved via adding a flag like, `is_inserted` in `TrieNode` and checking if its `True` in your method. I think similarly the method should be named, `is_inserted` and `is_present` should be kept untouched.